### ISSUE TITLE
Request: a higher-order function for 'valueAsString'

### ISSUE THREAD

**Issue by foodpoison** [TheJuki/KFormMaster#187]
## Description

Hi nice work on the library. I've got a request though, my specific usage is around using `FormPickerMultiCheckBoxElement` to work with a large number of items, I've got a big list of items that I am setting as filters for some data, and after selecting more than 4 or so items what gets displayed via `valueAsString` becomes meaningless to the user as it's too long. What I want is a way of writing custom logic there, and let's go with your fruits example for that field, if I select up to three fruits I still display results as list, but anything above that I'll instead show, say, `9 fruits selected`, or even `all fruits selected` or `empty basket`, etc, whatever.

Basically if I can have something like:
```kotlin
multiCheckBox<List<ListItem>>(MultiItems.ordinal) {
	...
	valueAsStringOverride = { values ->
		if (values.isEmpty()) "empty"
		else if (values == options) "all selected"
		else if (values.size > 5) "${value.size} fruit selected"
		else null
	}
	...
}
``` 
and the `var` declaration being something like:
```kotlin
var valueAsStringOverride: (T)-> String?
```
that if returned `null` it'll fallback to standard `valueAsString`

Something like that would be really nice.

**Comment by foodpoison** [comment]
Closing this as my code was merged.

**Comment by TheJuki** [comment]
I just released [6.5.4](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if this working for you.
